docs/modules/ROOT/pages/cops_rails.adoc

```diff
@@ -3959,7 +3959,7 @@ using `Object#present?` defined by Active Support.
 
 Interaction with `Style/UnlessElse`:
 The configuration of `NotBlank` will not produce an offense in the
-context of `unless else` if `Style/UnlessElse` is inabled. This is
+context of `unless else` if `Style/UnlessElse` is enabled. This is
```

Comment:
@thestelz The adoc is auto-generated. Can you update lib/rubocop/cop/rails/blank.rb and lib/rubocop/cop/rails/present.rb instead?

Reply:
@koic I didn't know that, sorry. I'll get 'em updated

Reply:
@koic sorry, missed that when approving.